index.rst

```diff
@@ -49,7 +49,7 @@ Configuration Interface Summary
 The standard interface for CSCs is defined in :lse:`209` and is the official reference/requirements document.
 This section provides a very brief overview of the key topics related to configuration that are described in further detail in the sections below.
 
-Configuration-related interactions with CSCs happens via two main events (``configurationsAvailable`` and ``configurationsApplied``) and a single parameter as part of the start command (``configurationOverride``).
+Configuration-related interactions with CSCs happens via two main events (``configurationsAvailable`` and ``configurationsApplied``) and a single parameter as part of the ``start`` command (``configurationOverride``).
```

Comment:
configurationsApplied --> configurationApplied

Reply:
There were three of these instances. All fixed.